include/oneapi/dnnl/dnnl_types.h

```diff
@@ -1020,6 +1020,7 @@ typedef enum {
     dnnl_AcdeB4b8a4b,
     dnnl_Ab4a,
     dnnl_Ab8a,
+    dnnl_Ab32a,
```

Comment:
please add this at the end of the list of tags (just before `dnnl_format_tag_last`). Otherwise, this breaks ABI backward compatibility.